### ISSUE TITLE
Unable to save class comment in recent Pharo 12

### ISSUE THREAD

**Issue by astares** [guillep/pharo-core#17379]
- download a fresh Pharo 12 image
- create a class Foo
- enter a class comment in Calpyso and save it

=> not saved, the class stays without a comment

Is this an issue with the keys or a bug in latest Pharo builds du to Microdown. 

Would be good if someone would be able to confim that issue. @jecisc ?

Side note: I'm using Windows here.

**Comment by Gabriel-Darbord** [comment]
I could not reproduce this on both Mac and Windows, class comments seem to work as intended.

**Comment by astares** [comment]
Seems related to *https://github.com/pharo-project/pharo/issues/17348*

**Comment by Ducasse** [comment]
I check today in the latest P12 I can see and edit the class comment. So I will close this issue.

**Comment by astares** [comment]
Confirm - also works in P13 now again

**Comment by astares** [comment]
@Ducasse 
Found the reasons and effect:  you start Pharo from the launcher by doube clicking and sometimes **accidentially the Pharo image is started twice** by the launcher. (Here I used Windows)

In such a situation both Pharo windows are displayed above each other and as they have the same size you might not notice.

Then the following can be seen
- in the **first started instance** it is possible to add class comments, this **image works normal**
- in the **instance that was started as a second one the class comments do not get accepted** (I guess this is because the source file might be locked by the first instance) - but **as no warning is shown, so it looks like this is a bug and class comments do not get accepted**

This is reproducible.
 - start launcher (on Windows)
 - double click an image to start
 - double click an image again to start a second instance
 - go to the window of the second instance, create a class and try to save a class comment. The class is accepted - but the comment is never changing

If the locked source file is really the cause then the second image should throw an exception and show an error message
